docs/src/modules/concepts/pages/grpc-vs-http-endpoints.adoc

```diff
@@ -0,0 +1,54 @@
+= Endpoints
+
+Endpoints components are how you expose your services to the outside world, be it another Akka service, external dependencies or client-facing applications. Two different types of endpoints are available: HTTP endpoints and gRPC endpoints.
+
+When designing APIs, choosing between HTTP and gRPC endpoints depends on the use case. This document outlines the key differences, strengths, and recommended usage for each approach.
+
+== Key Differences
+
+HTTP endpoints follow the traditional request-response model and are widely used for exposing APIs to external consumers. They use standard HTTP methods (GET, POST, PUT, DELETE, etc.) and typically exchange data in JSON format. This makes them accessible to web and mobile clients and easy to debug using common tools like browsers and API clients.
+
+gRPC endpoints use the HTTP/2 protocol and Protobuf for data serialization, providing efficient and structured communication between services. They support various types of RPC calls, including unary and streaming, and are designed for high-performance, low-latency interactions. gRPC is commonly used for internal service-to-service communication due to its strong typing and schema evolution capabilities.
+
+[cols="2,2,2"]
+|===
+| Aspect | HTTP Endpoints | gRPC Endpoints
+
+| Protocol | HTTP/1.1 or HTTP/2 | HTTP/2
+| Serialization | JSON (text-based) | Protobuf (binary, compact)
+| Performance | Higher latency due to JSON parsing | Lower latency with efficient binary serialization
+| Streaming | Limited support (SSE, WebSockets) | Native bidirectional streaming
+| Tooling & Debugging | Easy to inspect and test with browser and other tools | Requires specialized tools due to binary format
+| Browser Support | Works natively in browsers | Requires gRPC-Web proxy for frontend use
```

Comment:
Akka gRPC actually supports gRPC-web out of the box without requiring a proxy in front so that is not quite correct, maybe say only "Requires gRPC-web for use in browser"